components/dropzone/template.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Template
+page_title: DropZone Template
+description: Template for the Blazor DropZone content.
+slug: dropzone-template
+tags: telerik,blazor,dropzone,external,drag,drop,file,template,custom,content
+published: True
+position: 10
+---
+
+# DropZone Template
+
+The DropZone allows you to customize the default rendering of its content by using a `Template`. It is a `RenderFragment` so you can add any desired custom content - be that simple text, HTML elements or other components.
+
+The example below demonstrates placing the FileSelect inside the DropZone's `Template`. This allows you to still have a FileSelect component to handle the file selection but save space in the viewport. 
+
+All elements of the FileSelect but the Button are hidden as they are not needed for the specific scenario.
+
+>tip This result can also be achieved by hiding the whole FileSelect/Upload component and placing a custom button in the `Template` to open the `SelectFilesDialog`. See [live demo: DropZone - Template](https://demos.telerik.com/blazor-ui/dropzone/template)
+
+````CSHTML
+<TelerikDropZone Id="@DropZoneId">
+    <Template>
+        <TelerikFontIcon Icon="@FontIcon.Upload"/>
+        <span>Drag and drop files here</span>
+        <span>or</span>
+        <TelerikFileSelect DropZoneId="@DropZoneId"
+                           Class="custom-fileselect">
+        </TelerikFileSelect>
+    </Template>
+</TelerikDropZone>
```

Comment:
I personally would advise against showing such an example in the docs, as this seems to target a specific use-case, or even a missing functionality of ours(having the building blocks of the `Upload`, which are then reused in the DropZone, as essentially this is what is going on here), rather than a simple example of the `Template` of the component. Such an example does raises questions, such as why would one want to nest a dropzone into dropzone, and quite frankly looks more like a hack than an actual use-case(although it might indeed be a valid hack, which some clients might want to achieve).

Reply:
On a relevant note, [this PR](https://github.com/telerik/blazor-docs/pull/1326/files) includes a KB that explains why custom "Select Files" buttons don't work in Safari and provide a workaround.